scripts/ci/ensure-angular-dependencies-compatibility.js

```diff
@@ -0,0 +1,41 @@
+/**
```

Comment:
I pushed it here so we can see if the CI servers complain with another error. Feel free to push straight to this PR btw.